profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.module

```diff
@@ -45,11 +52,20 @@ function nexteuropa_token_ckeditor_menu() {
   $items = array();
   $items['nexteuropa/token-ckeditor/%'] = array(
     'page callback' => 'nexteuropa_token_ckeditor_ajax_callback',
-    'page arguments' => array(2, 3),
+    'page arguments' => array(2),
     'access callback' => 'nexteuropa_token_ckeditor_ajax_access_callback',
     'type' => MENU_CALLBACK,
     'delivery callback' => 'ajax_deliver',
   );
+
+  $items['nexteuropa/token-bean-ckeditor/%'] = array(
+    'page callback' => 'nexteuropa_token_ckeditor_bean_ajax_callback',
+    'page arguments' => array(2),
+    'access callback' => 'nexteuropa_token_ckeditor_bean_ajax_access_callback',
+    'type' => MENU_CALLBACK,
+    'delivery callback' => 'ajax_deliver',
+  );
+
```

Comment:
The business scenario looks to me crazy :). Ajax driven token empowered bean blocks inside wysiwig fields is a maintainance nightmare. But I guess there is no option, no way back, the business has spoken (info ?) @drupol